addons/binding/org.openhab.binding.neeo/src/main/java/org/openhab/binding/neeo/internal/type/NeeoConfigDescriptionProviderImpl.java

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.neeo.internal.type;
+
+import java.net.URI;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.Locale;
+import java.util.Map;
+
+import org.eclipse.smarthome.config.core.ConfigDescription;
+import org.osgi.service.component.annotations.Component;
+
+/**
+ * The implementation of {@link NeeoConfigDescriptionProvider} that simply maps the {@link ConfigDescription} to it's
+ * {@link URI}
+ *
+ * @author Tim Roberts - Initial Contribution
+ */
+@Component(service = { org.openhab.binding.neeo.internal.type.NeeoConfigDescriptionProvider.class,
```

Comment:
Similar as previously comment about omitting the package names in service and service name at all.